stats.js

```diff
@@ -0,0 +1,167 @@
+const apiKey = process.env.SOCRATA_KEY // Socrata API Key.
```

Comment:
Can these be pulled from secret manager and/or the parameter store when not running in dev?
Otherwise we'll need to have some one off lambda environment configuration in terraform, I believe. cc/ @pmcgrath & @dgonzalez to confirm.